### PR TITLE
Implement piano list UIs from CXM

### LINKED ISSUE #12: Implement piano list UI
Bring CXM's piano list UI into Pulsar, code exists in CXM repo at the moment, @jamesjacobs to send over.

Create:
* Documentation
* Lexicon examples

### PR DESCRIPTION
**Pull request by Stanton**
Documentation: https://jadu.gitbooks.io/pulsar/content/piano_list.html

Resolves #12 

Also fixes issue with Lexicon not loading module & pattern UIs

### CODE REVIEW

**Comment by Stanton** [comment]
Sass isn't as BEMish as it should be, but it's a straight port from CXM at the moment

File: stylesheets/_component.piano.scss [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+}
+
+.piano {
+    @include respond-min($screen-desktop) {
+        height: 75vh;
+        overflow-x: hidden;
+    }
+}
+
+.piano {
```
Comment (langy) [inline stylesheets/_component.piano.scss]: Could be merged with block directly above

File: views/lexicon/patterns/piano.html.twig [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+<div class="piano-wrapper">
+    <div class="piano">
+        <ul class="piano__items">
+
+        {% for i in 0..5 %}
+            <li class="piano__item{% if loop.first %} is-selected{% endif %}">
+                <a href="/piano_item_link">
+                    <div class="piano__item-header">
+                        <strong class="piano__user">
+                            <img src="https://upload.wikimedia.org/wikipedia/en/thumb/4/4d/Thenightisstillyoung.jpg/220px-Thenightisstillyoung.jpg" class="piano__avatar">
```
Comment (langy) [inline views/lexicon/patterns/piano.html.twig]: Perhaps remove `https:` so we don't get any nasties on plain old http

Comment (Stanton) [inline views/lexicon/patterns/piano.html.twig]: Good catch!

**Pull request merged**